src/yaml/builder.cr

```diff
@@ -130,11 +130,43 @@ class YAML::Builder
     yield.tap { end_mapping }
   end
 
-  def alias(anchor : String)
+  # Aliases *anchor*.
```

Comment:
Maybe "Emits an alias to the given \*anchor\*"? Same in the other method.

Reply:
@asterite What about `Emits the extend key, `<<`, and an alias to the given *anchor*.` for `#extend`?

Reply:
In the YAML spec it's called "merge": https://yaml.org/type/merge.html

I would probably rename `extend` to `merge`, say that it emits the scalar "<<" followed by an alias to the given anchor, and include a link to the spec.

Reply:
👍 I could go with that.